src/modules/tests/tests.effects.ts

```diff
@@ -205,6 +205,14 @@ export class TestsEffects {
         startTestAction.category === TestCategory.DE) {
         arrayOfActions.push(new InitializeVehicleChecksCatD(startTestAction.category));
       }
+      if (
+        startTestAction.category === TestCategory.F ||
+        startTestAction.category === TestCategory.G ||
+        startTestAction.category === TestCategory.H ||
+        startTestAction.category === TestCategory.K) {
+        // TODO Replace this with Home Test action
+        arrayOfActions.push(new InitializeVehicleChecksCatD(startTestAction.category));
+      }
```

Comment:
Transmission, Route Number and Independent Driving need to be defaulted in the effect - see miro for values